libgalois/src/analytics/pagerank/pagerank-pull.cpp

```diff
@@ -228,26 +224,54 @@ ComputePRTopological(Graph* graph, katana::analytics::PagerankPlan plan) {
 
 }  // namespace
 
+katana::Result<void>
+ExtractPagerank(
```

Comment:
This should be in the anonymous namespace since it's only used in this function. Also it should have a name that's specific to the algorithm that uses it. Because this isn't used for all of pagerank.

Reply:
Hmm. `ExtractScoreFromTopoGraph()` ?

Reply:
Aren't you extracting `value`?

Reply:
Value is pagerank score, ins't it. Ok then, `ExtractValueFromTopoGraph()` :-)